src/Microsoft.TestPlatform.CrossPlatEngine/Resources/Resources.resx

```diff
@@ -198,4 +198,7 @@
   <data name="TestDiscoveryCancelled" xml:space="preserve">
     <value>Discovery of tests cancelled.</value>
   </data>
+  <data name="AttachDebuggerToDefaultTestHostFailure" xml:space="preserve">
+    <value>Cannot attach the debugger to the default test host with process ID: {0}.</value>
```

Comment:
>Cannot attach the debugger to the default test host with process ID: {0}. [](start = 11, length = 73)

Can we add a why...

Reply:
Due to the way the Debugger APIs are designed the failure reason is not available for TP to display. There can be any number of reasons the debugger fails to attach, but we only get a negative pid in those cases.